model-optimizer/extensions/back/CropToStridedSlice.py

```diff
@@ -78,20 +84,24 @@ def replace_pattern(self, graph: Graph, match: [str, Node]):
             node.offset = self.list_to_ndarray(node.offset)
```

Comment:
All lines where the attributes of node `node` are updated are not needed, because the node `node` is replaced with StridedSlice. Please, cleanup.

Reply:
Done, now we use local variables instead of changing `node` attributes.